app/views/auth_source_externals/edit.html.erb

```diff
@@ -1,3 +1,14 @@
-<% title _("Edit External Authentication Source") %>
-
+<%= breadcrumbs(
+    items: [
+      {
+        caption: _("Authentication Sources"),
+        url: auth_sources_path
+      },
+      {
+        caption: _("Edit %s" % @auth_source_external.to_label),
```

Comment:
There can only be one external source, the caption should remain the same

Reply:
but you can rename it as you want, no?

Reply:
not easily right now